app/api/users.js

```diff
@@ -0,0 +1,28 @@
+var bodyParser = require( 'body-parser' );
+var userRepository = require( '../repositories/user.js' );
+
+/**
+ *  This is the main part in the user api
+ *
+ *  From here all methods connected with users will be executed
+ *
+ * @param app
+ */
+module.exports = function( app ) {
+  app.use( bodyParser.json() );
+  app.use( bodyParser.urlencoded( {extended: true} ) );
+
+  // When a new user needs to be registered
+  app.post( '/api/users', function( req, res ) {
+
+    //Get the user parameters from the request's body and execute the create user method from
```

Comment:
You can use multiline comment here.
Also your comment has to **generally** describe the functionality, instead of interpret the code literally. Image if in the future changes you stop using userRepository, then you have to change your comment too.

For example you can documentate your createUser functionality as follow:
_Create user by the passed request's parameters._